vyper/venom/ir_node_to_venom.py

```diff
@@ -114,6 +114,8 @@
 
 # convert IRnode directly to venom
 def ir_node_to_venom(ir: IRnode) -> IRContext:
+    _ = ir.unique_symbols  # run unique symbols check
```

Comment:
This seems rather strange at a quick glance

Reply:
unique_symbols performs a sanity check